modules/polaris/src/main/ts/ephox/polaris/pattern/Safe.ts

```diff
@@ -1,9 +1,10 @@
 import { PRegExp } from './Types';
 import * as Unsafe from './Unsafe';
+import { Regex } from '@ephox/katamari';
 
 /** Escapes regex characters in a string */
 const sanitise = (input: string): string => {
```

Comment:
This could be simplified/reduced to just `const sanitise = Regex.escapeRegExp;`

Reply:
That would be a breaking change, which seems pointless for this small change. Maybe we should just leave a TODO here that we should remove this in the next major instead?

Reply:
Good call — I just looked at the file path and hoped it wasn't exported.